portfolio/src/main/java/com/google/sps/servlets/CheckStatus.java

```diff
@@ -21,8 +23,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
         out.println(new Gson().toJson(true));
+<<<<<<< HEAD
+<<<<<<< HEAD
+    }
+    else {
+=======
     }else{
+>>>>>>> 1e9b006429eac588504484c5c9e9fe1332bdf466
+=======
+    }
+    else {
+>>>>>>> 59e25543aa73e9ebc121f9f644e29d02bc0f27ae
```

Comment:
Same idea here/

Also make sure your else statement falls in the same line as the closing brace before:

```
} else {
```

Rather than 

```
}
else {
```